nab/util.py

```diff
@@ -173,7 +173,7 @@ def absoluteFilePaths(directory):
   for dirpath,_,filenames in os.walk(directory):
     filenames = [f for f in filenames if not f[0] == "."]
     for f in filenames:
-      yield os.path.abspath(os.path.join(dirpath, f))
+      yield os.path.abspath(os.path.normpath(os.path.join(dirpath, f)))
```

Comment:
Part of some old modifications during testing. Reverting the change back works, considering this just handling a directory and file joining.